index.js

```diff
@@ -0,0 +1,91 @@
+var analyze = require('./lib/analyze-raptor');
+var Promise = require('promise');
+var fs = require('fs');
+
+var metric = 'visuallyLoaded';
+
+var readFile = function(dataFile, appContext, appEntryPoint) {
+  console.log('Processing %s', dataFile);
+  console.log('App (context: %s, entrypoint: %s)', appContext, appEntryPoint);
+  return new Promise(function(resolve, reject) {
+    fs.readFile(dataFile, { encoding: 'utf8' }, function(err, content) {
+      if (err) {
+        return reject(err);
+      }
+      var data = content.split('\n');
```

Comment:
Insert 1 line above `var data`. Also 1-line the variable into: `var data = JSON.parse(content.split('\n'));` instead of reassigning the value.